packages/relay/src/lib/config/hbarSpendingPlanConfigService.ts

```diff
@@ -108,16 +109,19 @@ export class HbarSpendingPlanConfigService {
    * @private
    */
   private loadSpendingPlansConfig(): SpendingPlanConfig[] {
-    const configPath = findConfig(this.SPENDING_PLANS_CONFIG_FILE);
-    if (!configPath || !fs.existsSync(configPath)) {
-      this.logger.trace(`Configuration file not found at path "${configPath ?? this.SPENDING_PLANS_CONFIG_FILE}"`);
-      return [];
+    const configPath = findConfig(this.SPENDING_PLANS_CONFIG);
```

Comment:
nit: to make the variable name more accurate
```suggestion
    const spendingPlanConfig = findConfig(this.SPENDING_PLANS_CONFIG);
```